### ISSUE TITLE
10.0.26052.1000

### ISSUE THREAD

**Issue by udygct** [stascorp/rdpwrap#2481]
Termsrv.dll
[termsrv.zip](https://github.com/stascorp/rdpwrap/files/14223303/termsrv.zip)

![image](https://github.com/stascorp/rdpwrap/assets/110302290/0a78fd28-5870-424f-9c15-4cfa35b81555)

**Comment by sashaqwert** [comment]
Please, attach your `termsrv.dll`

**Comment by udygct** [comment]
I tried this twice on version 26052. After disconnect the RDP once, it showed "Remote Procedure Call Failed" and I could not log in again. I reset my computer and the results are the same.

**Comment by sashaqwert** [comment]
https://github.com/sebaxakerhtc/rdpwrap.ini/commit/bd193dee2549976d3c85fae0e1e90ee08a990ebf

**Comment by sashaqwert** [comment]
@udygct 

> I tried this twice on version 26052. After disconnect the RDP once, it showed \"Remote Procedure Call Failed\" and I could not log in again. I reset my computer and the results are the same.

Did this appear after today's INI update?

**Comment by udygct** [comment]
Yeah, this occurred after today’s INI update. I think Microsoft made some changes to the RDP in version 26052, as the interface’s appearance differs from the previous version.

**Comment by sashaqwert** [comment]
> Microsoft made some changes to the RDP in version 26052,

@sebaxakerhtc

**Comment by sebaxakerhtc** [comment]
> > Microsoft made some changes to the RDP in version 26052,
> 
> @sebaxakerhtc

https://github.com/sebaxakerhtc/rdpwrap.ini/issues/345#issuecomment-1941094719 All works fine

**Comment by udygct** [comment]
I tried this again on a Hyper-V virtual machine with a clean install of version 26052 and still got stuck at the login page.
![image](https://github.com/stascorp/rdpwrap/assets/110302290/0d6f3494-3002-471f-8238-165938dca883)
After rebooting the VM and enter the password to login, it shows the "The remote procedure call failed" and then it just stock here and I cannot log in to my VM anymore.
![image](https://github.com/stascorp/rdpwrap/assets/110302290/0e926adf-ad92-43a2-81e6-d614f3051c8b)
I am pretty sure there is something different in 26052.
![image](https://github.com/stascorp/rdpwrap/assets/110302290/9c1192a2-561f-4ea6-812e-534d677e9fd4)

**Comment by sashaqwert** [comment]
> I tried this again on a Hyper-V virtual machine with a clean install of version 26052 and still got stuck at the login page.

You need convert your Microsoft Account to local account.

> After rebooting the VM and enter the password to login, it shows the "The remote procedure call failed" and then it just stock here and I cannot log in to my VM anymore.

Bug... Try boot Windows in safe mode.

**Comment by udygct** [comment]
It does work! Booting Windows in safe mode does allow me to log in but the problem still here after rebooting without using safe mode.... Is there another way to fix this bug without using safe mode?

**Comment by sashaqwert** [comment]
> It does work! Booting Windows in safe mode does allow me to log in but the problem still here after rebooting without using safe mode.... Is there another way to fix this bug without using safe mode?

@sebaxakerhtc

**Comment by udygct** [comment]
termsrv.dll
[termsrv(26058).zip](https://github.com/stascorp/rdpwrap/files/14300587/termsrv.26058.zip)
![image](https://github.com/stascorp/rdpwrap/assets/110302290/93ef453e-8f2b-44e8-a8e3-681ab875e4e5)

**Comment by sebaxakerhtc** [comment]
> > It does work! Booting Windows in safe mode does allow me to log in but the problem still here after rebooting without using safe mode.... Is there another way to fix this bug without using safe mode?
> 
> @sebaxakerhtc

Why are you pinging me :) ?
This version works fine - the problem just on his side...

**Comment by sebaxakerhtc** [comment]
> termsrv.dll [termsrv(26058).zip](https://github.com/stascorp/rdpwrap/files/14300587/termsrv.26058.zip) <img alt="image" width="251" src="https://private-user-images.githubusercontent.com/110302290/305174294-17d5b6fd-eeed-417e-ab6f-677fa9021961.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgwMjM5NDYsIm5iZiI6MTcwODAyMzY0NiwicGF0aCI6Ii8xMTAzMDIyOTAvMzA1MTc0Mjk0LTE3ZDViNmZkLWVlZWQtNDE3ZS1hYjZmLTY3N2ZhOTAyMTk2MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxNVQxOTAwNDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hODYxZDM3MmE5NTk1MzVjODJmM2Q4ODhjMWMwNzNmMzFlODgwOGRkOTIxY2FmODYzYzI4OTEyYTcyYzFkYWIwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.K7YKTxc3DByxepHW4bFXrVrNekHwYN-S7vOAiMhZ1y4">

New issue for each version! Rename it back.

**Comment by sashaqwert** [comment]
Reopened. Not included in official INI by Stas'M...

**Comment by sashaqwert** [comment]
> Why are you pinging me :) ?
> This version works fine - the problem just on his side...

Confirmed :) https://github.com/stascorp/rdpwrap/issues/2488#issuecomment-1947895498 --> ignoring...

**Comment by udygct** [comment]
Thanks to @sashaqwert and @sebaxakerhtc for helping. At least now I can recover my data by deleting the rdpwrap.dll in safe mode.

**Comment by sebaxakerhtc** [comment]
@binarymaster

**Comment by sashaqwert** [comment]
Reopened. Not included in official INI by Stas'M :)

**Comment by udygct** [comment]
The problem is fixed https://github.com/stascorp/rdpwrap/issues/2488#issuecomment-1958862542
